src/components/SearchQuery/search-query.jsx

```diff
@@ -14,24 +14,24 @@ const Results = connectStateResults(({ searchState }) =>
   (searchState && searchState.query ? (
     <div>
       <CustomHits />
+      <Pagination />
     </div>
   ) : (
-    <div>No query</div>
+    <div> </div>
```

Comment:
Open to suggestions about placeholder text here, but I think it looks okay without anything there.

Reply:
I thought the "No query" was clunky, just copied it from the example.  I don't think we need anything there

Reply:
Great!